src/components/Modals/LedgerOpenApp/LedgerOpenAppModal.tsx

```diff
@@ -50,12 +51,17 @@ export const LedgerOpenAppModal = ({ chainId, onCancel }: LedgerOpenAppModalProp
         </ModalHeader>
         <ModalBody>
           <VStack spacing={2}>
-            <RawText color='whiteAlpha.600'>
+            <RawText color='whiteAlpha.600' textAlign='center'>
               {translate('ledgerOpenApp.description', {
                 appName,
               })}
             </RawText>
-            <Spinner size='lg' />
+            {isSigning ? (
+              <RawText color='whiteAlpha.600' textAlign='center'>
```

Comment:
suggestion: wrap me in `<Alert status='warning' />` to make this more obvious

Reply:
![image](https://github.com/user-attachments/assets/f28ddc4f-18d3-4be8-b433-3688e0b12ac0)

Here we go [df93da2](https://github.com/shapeshift/web/pull/7601/commits/df93da290cddfc301c2970d8eceea07d59a6453b)